src/OnePlusBot/Base/CommandHandler.cs

```diff
@@ -39,24 +39,29 @@ private async Task HandleCommandAsync(SocketMessage messageParam)
             int argPos = 0;
 
             // Getting the bot's guilds and setting variables for the OnePlus Guild and their set-ups channel.
-            IReadOnlyCollection<SocketGuild> guilds = _bot.Guilds;
-            SocketGuild oneplusGuild = guilds.FirstOrDefault(x => x.Id == 378969558574432277);
-            SocketGuildChannel setupsChannel = oneplusGuild.Channels.FirstOrDefault(x => x.Id == 473051502022361119);
-            var channel = messageParam.Channel as ITextChannel;
-
-
-            if (channel.GuildId == oneplusGuild.Id)
+            try
             {
-                if (messageParam.Channel.Id == setupsChannel.Id)
+                IReadOnlyCollection<SocketGuild> guilds = _bot.Guilds;
+                SocketGuild oneplusGuild = guilds.FirstOrDefault(x => x.Id == 500105004082790400);
```

Comment:
This will make it work on your server and no where else, the ID is being checked for your server and your server channel.